src/ray/raylet/scheduling/cluster_task_manager_test.cc

```diff
@@ -190,18 +192,18 @@ class ClusterTaskManagerTest : public ::testing::Test {
     ASSERT_TRUE(task_manager_.waiting_tasks_index_.empty());
     ASSERT_TRUE(task_manager_.waiting_task_queue_.empty());
     ASSERT_TRUE(task_manager_.infeasible_tasks_.empty());
+    ASSERT_TRUE(task_manager_.executing_task_args_.empty());
```

Comment:
Should we also check pinned_task_args_bytes is zero?

Reply:
Ah thanks, I missed that one.